src/main/java/frc/robot/commands/IntakeSequence.java

```diff
@@ -11,15 +14,25 @@ public class IntakeSequence extends SequentialCommandGroup {
   private Intake intake;
   private Lift lift;
 
-  private InstantCommand deployIntake = new InstantCommand(intake::deploy, intake);
-  private InstantCommand runIntake = new InstantCommand(intake::intakeGamePiece, intake);
+  // deploy intake for specified amount of time
+  private ParallelRaceGroup deployIntake =
+      new RunCommand(intake::deploy, intake).withTimeout(Cal.Intake.AUTO_CLAMP_WAIT_TIME_SECONDS);
+
+  // run intake, move lift to intake position, and open the grabber, until the robot sees an object
+  // and the lift is in position
+  private RunCommand runIntake = new RunCommand(intake::intakeGamePiece, intake);
   private InstantCommand liftToIntakePos =
       new InstantCommand(() -> lift.setDesiredPosition(LiftPosition.GRAB_FROM_INTAKE), lift);
   private InstantCommand openGrabber = new InstantCommand(lift::openGrabber, lift);
   private ParallelRaceGroup intakePrep =
-      new ParallelRaceGroup(runIntake, liftToIntakePos, openGrabber).until(lift::readyToIntake);
+      new ParallelCommandGroup(runIntake, liftToIntakePos, openGrabber).until(lift::readyToIntake);
+
   private InstantCommand closeGrabber = new InstantCommand(lift::closeGrabber, lift);
+  
   private InstantCommand unclampIntake = new InstantCommand(intake::unclampIntake, intake);
+
+  // this does not need a timeout even though it is a longer action, because it is the final action
+  // in the sequence
   private InstantCommand liftToStartPos =
```

Comment:
Perhaps add a comment describing what `liftToStartPos` does in addition to the original comment provided?